src/python/pants/backend/python/util_rules/pex_from_targets_test.py

```diff
@@ -443,3 +461,90 @@ def test_issue_12222(rule_runner: RuleRunner) -> None:
     result = rule_runner.request(PexRequest, [request])
 
     assert result.requirements == PexRequirements(["foo"], constraints_strings=constraints)
+
+
+class ResolveMode(Enum):
+    run_against_entire_lockfile = "run_against_entire_lockfile"
```

Comment:
If I understand correctly, we are only testing this using requirement constraints. It also applies with resolves, regardless of pex versus poetry format. Should we test that too?

Reply:
Hm, good point. Yea, will fix.